modules/Bio/EnsEMBL/Production/Pipeline/PipeConfig/Base_conf.pm

```diff
@@ -24,7 +24,7 @@ use warnings;
 use base ('Bio::EnsEMBL::Hive::PipeConfig::EnsemblGeneric_conf');
 
 use Bio::EnsEMBL::Hive::PipeConfig::HiveGeneric_conf;
-use Bio::EnsEMBL::Hive::Version 2.5;
+use Bio::EnsEMBL::Hive::Version v2.5;
```

Comment:
Why 2.5? Should we use 2.7 for slurm? Do we need version check at all? Sorry 🙈

Reply:
This change is neutral w.r.t. the question of whether a version check is needed.

It only aims to make clear to the version check — by using a v-string — that `v2.5` is a version which is lower than the version of Hive we're currently using (`v.2.7.0`).

Reply:
Thank you!